### ISSUE TITLE
Can't change the langauge to call redact in ImageRedactorEngine to use another language in tesseract.

### ISSUE THREAD

**Issue by rob-nn** [microsoft/presidio#1274]
**Describe the bug**

**To Reproduce**
Create a py file with the code:

```
from presidio_image_redactor import ImageRedactorEngine
from PIL import Image
engine = ImageRedactorEngine()
# Do not forget to create the tessdata with the data for the language wished
ocr_kwargs = {'config':"--tessdata-dir tessdata", 'lang':"por"}
image = Image.open('teste.png')
redacted_image = engine.redact(image, (0, 0, 0), language='pt', ocr_kwargs=ocr_kwargs)
```
**Expected behavior**
Run without troubles

**Screenshots**
```
File /usr/local/lib/python3.11/site-packages/presidio_image_redactor/image_analyzer_engine.py:53, in ImageAnalyzerEngine.analyze(self, image, ocr_kwargs, **text_analyzer_kwargs)
     50 # Analyze text
     51 text = self.ocr.get_text_from_ocr_dict(ocr_result)
---> 53 analyzer_result = self.analyzer_engine.analyze(
     54     text=text, language="en", **text_analyzer_kwargs
     55 )
     56 bboxes = self.map_analyzer_results_to_bounding_boxes(
     57     analyzer_result, ocr_result, text
     58 )
     59 return bboxes

`TypeError: presidio_analyzer.analyzer_engine.AnalyzerEngine.analyze() got multiple values for keyword argument 'language'
```

**Comment by rob-nn** [comment]
The bug occurs when the parameter language is passed:
Exemple:
```
image = PIL.image.new(mode="RGB", size=(200, 200))
redacted = ImageAnalyzerEngine().analyze(get_mock_png, language='en')
```

Produce the same error: 
`TypeError: presidio_analyzer.analyzer_engine.AnalyzerEngine.analyze() got multiple values for keyword argument 'language'`

**Comment by omri374** [comment]
Fixed in #1275
